Description: Edit Makefile to support $DESTDIR and FHS as well as create its own directories

--- a/Makefile
+++ b/Makefile
@@ -1,18 +1,20 @@
 CFLAGS = -DSHAPE -Wall -Werror
 LDLIBS = -lXext -lX11
-BIN = /usr/bin/X11
+BIN = $(DESTDIR)/usr/bin
 
-MANDIR = /usr/man/man1
+MANDIR = $(DESTDIR)/usr/share/man/man1
 MANSUFFIX = 1
 
 all: 9wm
 
 9wm: 9wm.o event.o manage.o menu.o client.o grab.o cursor.o error.o
 
-install: 9wm
+install: 9wm install.man
+	mkdir -p $(BIN)
 	cp 9wm $(BIN)/9wm
 
 install.man:
+	mkdir -p $(MANDIR)
 	cp 9wm.man $(MANDIR)/9wm.$(MANSUFFIX)
 
 $(OBJS): $(HFILES)
